tests/integration/cli/up_test.py

```diff
@@ -9,6 +10,13 @@ def setUp(self):
         busybox_single_app_bundle_fixture(num_bundles=2)
         self.run_command('bundles activate busyboxa busyboxb')
 
+    def tearDown(self):
+        try:
+            self.run_command('stop')
+        except Exception:
+            pass
+        super(TestUpCLI, self).tearDown()
```

Comment:
Explicitly doing the stop makes the `up` integration tests much faster. Without this, we end up doing slow teardowns at the beginning of each subsequent test, which takes 10 seconds per container. With this, it's about 1 second per container.